main_test.go

```diff
@@ -53,6 +53,61 @@ func TestSendMetrics(t *testing.T) {
 	assert.NoError(err)
 }
 
+func TestSendMetricsHostStripping(t *testing.T) {
+	var hostStrippingDataSet = []struct {
+		entityName string
+		expectedBody string
+		stripHost    bool
+	}{
+		{
+			entityName: "sensu.company.com",
+			expectedBody: `answer,foo=bar,sensu_entity_name=sensu.company.com value=42`,
+			stripHost: true,
+		},
+		{
+			entityName: "answer.company.com",
+			expectedBody: `answer,foo=bar,sensu_entity_name=answer.company.com value=42`,
+			stripHost: true,
+		},
+		{
+			entityName: "prod-eu-db01.company.com",
+			expectedBody: `answer,foo=bar,sensu_entity_name=prod-eu-db01.company.com value=42`,
+			stripHost: true,
+		},
```

Comment:
We're missing a test case where `stripHost:  true` and `!strings.HasPrefix(point.Name, event.Entity.Name)`.

Reply:
I added the metricName field at the test dataset so we can trigger `!strings.HasPrefix(point.Name, event.Entity.Name)`. I hope every case is now covered.